_pages/javascript/style.md

```diff
@@ -8,7 +8,7 @@ We recommend combining [Prettier](https://prettier.io) with the
 [Airbnb JavaScript style guide](https://github.com/airbnb/javascript) plugins
 for [eslint](https://eslint.org).
 
-Maintaining stylistic consistency across 18F's code helps lower the barrier to
+Maintaining stylistic consistency across TTS' code helps lower the barrier to
```

Comment:
The possessive here caught my attention, I suppose since it ends with "S" it's unclear if "TTS's" would be correct. Might it be fine without the apostrophe at all?

```suggestion
Maintaining stylistic consistency across TTS code helps lower the barrier to
```

Reply:
+1 for removing it altogether – I saw so many different guides and even now couldn't tell you what is correct for initialisms whose expansion ends in a plural noun in American English 😄